tutorials/forest-cover-loss-estimation/index.md

```diff
@@ -0,0 +1,173 @@
+
+// Load country features from Large Scale International Boundary (LSIB) dataset.
+var countries = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017');
+var selected = countries.filter(ee.Filter.eq('country_na', ee.String(country)));
+```
+
+Depending on the location of the country, the number of pixels that make up for 0.5 ha would differ. You can adjust this by calculating the actual minimum areas you were using (see step 6 below).
+
+### Tree cover
+
+Currently, Google Earth Engine has several tree cover datasets in the catalogue, including the Global Forest Change (GFC) (year 2000) and GLCF: Landsat Tree Cover Continuous Fields (2000, 2005, and 2010). Here, we use the Global Forest Change dataset. 
+```js
+var gfc2018 = ee.Image('UMD/hansen/global_forest_change_2018_v1_6');
+```
+
+1. Select 'treecover2000' in the Global Forest Change dataset
+```js
+var canopyCover = gfc2018.select(['treecover2000']);
+```
+2. Apply the minimum canopy cover percentage (e.g. greater than or equal to 10%). Use `selfMask()` to mask value zero
+```js
+var canopyCover10 = canopyCover.gte(cc).selfMask();
+```
+3. Apply the minimum area requirement using `connectedPixelCount` (e.g. greater than or equal to 6 pixels). Note that if no kernel is passed to `connectedPixelCount`, 8 neighbor adjacency is used to determine connectivity
+```js
+// Use connectedPixelCount() to get contiguous area.
+var contArea = canopyCover10.connectedPixelCount();
+// Apply the minimum area requirement. 
+var minArea = contArea.gte(pixels).selfMask();
+```
+4. Scale the results in nominal value based on to the dataset's projection to display on the map
+```js
+var prj = gfc2018.projection();
+var scale = prj.nominalScale();
+Map.addLayer(minArea.reproject(prj.atScale(scale)), {
+    palette: ['#96ED89']
+}, 'tree cover: >= min canopy cover & area (light green)');
+```
+5. Calculate the tree cover area (ha)
+```js
+var forestArea = minArea.multiply(ee.Image.pixelArea()).divide(10000);
+var forestSize = forestArea.reduceRegion({
+    reducer: ee.Reducer.sum(),
+    geometry: selected.geometry(),
+    scale: 30,
+    maxPixels: 1e13
+});
+print('Year 2000 tree cover (ha) \nmeeting minimum canopy cover and \nforest area thresholds \n ', forestSize.get('treecover2000'));
+```
+6. Calculate the actual average minimum forest area used (ha) 
+```js
+var pixelCount = minArea.reduceRegion({
+    reducer: ee.Reducer.count(),
+    geometry: selected.geometry(),
+    scale: 30,
+    maxPixels: 1e13
+});
+var onePixel = forestSize.getNumber('treecover2000').divide(pixelCount.getNumber('treecover2000'));
+var minAreaUsed = onePixel.multiply(pixels);
+print('Minimum forest area used (ha)\n ', minAreaUsed);
+```
+The GFC dataset uses 30x30m pixels. Therefore, 6 pixels (>5,000/(30x30)) is used for this exercise. This can be used for countries near the equator. For countries in higher or lower latitudes, you would need to increase the number of pixels. 
```

Comment:
Please mention why, e.g., due to difference in resolutions of the source dataset at different latitudes.

Reply:
This may go into GIS 101 - perhaps and hopefully users are familiar?  If not, can you suggest what kind of explanation may be appropriate in the tutorial?

Reply:
I think this gives enough context for users to be able to search more if they're interested. Looks good!